docs/gateway-configuration/wifi-configuration-8021x.md

```diff
@@ -0,0 +1,47 @@
+# Wi-Fi 802.1x Configuration
+
+Enterprise Wi-Fi is currently only supported on gateways running our [generic profiles](/getting-started/install-kura/#installer-types). The following is a list of currently supported 802.1x configurations.
+
+ - TTLS-MSCHAPv2
+ - PEAP-MSCHAPv2
+ - TLS-EAP
+
+# Setup TTLS/PEAP-MSCHAPv2
+ 1. Set up gateway Wi-Fi as described in the [Wi-Fi configuration guide](/gateway-configuration/wifi-configuration/#wireless-configuration).
+ 2. Ensure Wireless Security is set to `WPA2/WPA3-Enterprise`
+    ![Alt text](images/802-1x-images/wifi-enterprise-enum.png)
+ 3. select the 802.1x tab
+ 4. Set Enteprise EAP -> `TTLS/PEAP`
+ 5. Set Inner Authentication -> `MSCHAPV2`
+ 6. Set `Identity (Username)` 
+ 7. Set `Password`
+ 6. Press 'Apply'
+
+The configuration should look like the following:
+![Alt text](images/802-1x-images/wifi-ttls.png)
+
+# Setup TLS-EAP
+To connect via TLS-EAP you will need the following items in unencrypted PEM format:
+    - Certificate Authority (CA) Certificate
+    - Client Certificate + Private Key (PKCS8)
+
+Enrolling secrets in the Keystore service
```

Comment:
I would add a link to the existing documentation: https://eclipse.github.io/kura/docs-release-5.3/gateway-configuration/keystores-management/